tt_metal/jit_build/build.cpp

```diff
@@ -197,6 +197,11 @@ void JitBuildState::finish_init() {
     // Note the preceding slash which defies convention as this gets appended to
     // the kernel name used as a path which doesn't have a slash
     this->target_full_path_ = "/" + this->target_name_ + "/" + this->target_name_ + ".elf";
+
+    if (not this->is_fw_) {
```

Comment:
! is more readable -- this isn't python :)

Reply:
Personally, I prefer "!" but we've adopted "not" for booleans